packages/experimental/Button/src/CompoundButton/SPEC.md

```diff
@@ -0,0 +1,81 @@
+# Compound Button
+
+## Background
+
+The `CompoundButton` is a `Button` that is designed to show an extra line of text.
+
+## Sample Code
+
+Basic examples:
+
+```jsx
+<CompoundButton secondaryContent="A second line">Text</CompoundButton>
+<CompoundButton icon={{ svgSource: { uri: 'https://www.example.com/test.svg', viewBox: '0 0 100 100' } }} secondaryContent="A second line" />
+<CompoundButton icon={{ svgSource: { uri: 'https://www.example.com/test.svg', viewBox: '0 0 100 100' } }} secondaryContent="A second line">Text</CompoundButton>
+<CompoundButton appearance="primary" secondaryContent="A second line">Text</CompoundButton>
+<CompoundButton disabled secondaryContent="A second line">Text</CompoundButton>
+<CompoundButton size="small" secondaryContent="A second line">Text</CompoundButton>
+```
+
+More examples in the Tester App.
+
+## Variants
+
+Variant options are the same as the base `Button` component. This section will note main differences in design from the the base `Button` component.
+
+### Icon
+
+The space for the icon is larger in a `CompoundButton`. The `CompoundButton` is designed so that it can hold an illustration in the place of an icon.
+
+## API
+
+### Slots
+
+- `root` - The outer container representing the `Button` itself that wraps everything passed via the `children` prop.
+- `icon` - If specified, renders an `icon` either before or after the `children` as specified by the `iconPosition` prop.
+- `loader` - If specified, renders a `loader` before the `icon` and `children` while the `loading` flag is set to `true`.
```

Comment:
I remember comment from Sunmin that there won't be icons when we have a loading button: "Loading button will not have any icon next to it. It'll always have spinner + text 'loading'"

Reply:
Thanks, I'll update the text